CHANGELOG.md

```diff
@@ -1,4 +1,79 @@
 # Changelog
+## Friday 9th September 2022
+### Voice Cloner SDK
+* [https://github.com/aflorithmic/apiaudio-python/pull/121](Voice Cloner SDK) - 
+* SDK's readability and the accuracy of API's representations (object members annotations in IDEs!),
```

Comment:
<3
Next SDK V2 with Classes

Reply:
That's about to come it's not rolled out. There should be a note here like what's cooking or coming soon or whatever but it's not a change that's been introduced yet.

And SDK v2 will come even later than that.